config/environments/production.rb

```diff
@@ -93,4 +93,6 @@
   # config.active_record.database_selector = { delay: 2.seconds }
   # config.active_record.database_resolver = ActiveRecord::Middleware::DatabaseSelector::Resolver
   # config.active_record.database_resolver_context = ActiveRecord::Middleware::DatabaseSelector::Resolver::Session
+
+  config.auth[:shared_key] = ENV.fetch('PRODUCTION_JWE_SHARED_KEY')
```

Comment:
this should be `JWE_SHARED_KEY` otherwise we have to change the key for every environment and the environment needs to know what environment its in

Reply:
will fix